client_cli/pytests/src/client_cli/configuration.py

```diff
@@ -0,0 +1,124 @@
+"""
+This module provides a Config class to manage Iroha network configuration.
+"""
+
+import json
+import os
+import random
+from urllib.parse import urlparse
+
+
+class Config:
+    """
+    Configuration class to handle Iroha network configuration. The class is responsible for reading
+    the configuration file, updating the TORII_API_URL with a random port number from the specified
+    range, and providing access to the updated TORII_API_URL.
+
+    :param path_config_client_cli: The path to the configuration file.
+    :type path_config_client_cli: str
+    :param port_min: The minimum port number for the TORII_API_URL.
+    :type port_min: int
+    :param port_max: The maximum port number for the TORII_API_URL.
+    :type port_max: int
+    """
+    def __init__(self, path_config_client_cli, port_min, port_max):
+        if not os.path.exists(path_config_client_cli):
+            self.create_default_config(path_config_client_cli)
```

Comment:
I wouldn't do this. If there is no config file I would just throw an exception

Reply:
> I'd rather the constructor not load anything from disk at all. This is not the level at which the decision to load a file should be made.
> 
> Pep 20 [point two](https://peps.python.org/pep-0020/), "explicit is better than implicit".

fixed